charts/4allportal/templates/samba/secret.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if $.Values.samba.enabled }}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: {{ include "common.secrets.name" (dict "existingSecret" (dict) "defaultNameSuffix" "samba" "context" $) }}
+  namespace: {{ .Release.Namespace }}
+stringData:
+  users: |-
+    {
+      "samba-container-config": "v0",
+      "users": {
+        "all_entries": [
+          {
+            "name": "admin",
+            "password": "CjXYjwF7kqLaf2NbAqTSdAUqS"
```

Comment:
Auto generate or set via value, unconfigurable default via HelmChart is a massive security risk

Reply:
Auto generation would also mean regenerating admin password per restart